### PR TITLE
NAVQP-227: implemented auto-focus for OV5640

### PR DESCRIPTION
**Pull request by dnbazhenov**

### CODE REVIEW

File: drivers/media/i2c/ov5640.c [hunk 1]
```diff
@@ -3143,6 +3197,144 @@ static int ov5640_set_framefmt(struct ov5640_dev *sensor,
...
+static int ov5640_copy_fw_to_device(struct ov5640_dev *sensor,
+					const struct firmware *fw)
+{
+	struct i2c_client *client = sensor->i2c_client;
+	const u8 *data = (const u8 *)fw->data;
+	u8 fw_status;
+	int i;
+	int ret;
+
+	// Putting MCU in reset state
```
Comment (avyurtsev) [inline drivers/media/i2c/ov5640.c]: Please convert c++ comment to c

File: drivers/media/i2c/ov5640.c [hunk 2]
```diff
@@ -3143,6 +3197,144 @@ static int ov5640_set_framefmt(struct ov5640_dev *sensor,
...
+	ret = ov5640_write_reg(sensor, OV5640_REG_SYS_RESET00, 0x20);
+	if (ret)
+		return ret;
+
+	// Write firmware
+	for (i = 0; i < fw->size / sizeof(u8); i++)
```
Comment (avyurtsev) [inline drivers/media/i2c/ov5640.c]: It seems that sizeof(u8) is not needed here.

Comment (dnbazhenov) [reply drivers/media/i2c/ov5640.c]: Removed

File: drivers/media/i2c/ov5640.c (hunk 1 above)
Comment (dnbazhenov) [reply drivers/media/i2c/ov5640.c]: Converted

**Review by avyurtsev — APPROVED** [review]

**Pull request merged**